class.jetpack.php

```diff
@@ -7302,17 +7296,18 @@ public static function is_gutenberg_available() {
 	 * @return void
 	 */
 	public static function load_jetpack_gutenberg() {
+		if ( ! Jetpack::is_active() ) {
```

Comment:
Noting that while this works for now, we'll need to add more conditions here, as per #10198